www_src/pages/map/map.less

```diff
@@ -121,28 +83,33 @@
     bottom: 0;
     width: 100%;
     padding-bottom: 10px;
+    align-items: center;
 
     button {
-      margin: 5px;
-      font-size: 20px;
-      color: white;
-      border: none;
+      margin: 5px 10px;
       border-radius: 50%;
-      background: #ccc;
+      background: #FAFAFA;
+      border: 0px solid rgba(35,31,32,0.02);
+      box-shadow: 0px 3px 3px 0px rgba(0,0,0,0.16), 0px 3px 3px 0px rgba(0,0,0,0.23);
```

Comment:
This looks way too harsh to me, although ricardo's gonna do some work on the design and file a follow up